Webex Connect Flows/v3.2/Sample/Usage Of Surveys in Flows/README.md

```diff
@@ -0,0 +1,77 @@
+- When a flow administrator needs to send survey messages or survey links to customers, these flows can be utilized.
+
+## Setting up the flow with a survey node.
+
+### Setting up the Survey Node
+
+<br><img width="800" alt="Survey" src="../../images/Survey.png"><br>
+
+1. Select the Method Name as Create a New Survey Token for a Questionnaire.
+2. Select the necessary Node Runtime Authorization from the dropdown.
+3. Click "Fetch All Questionnaires" to retrieve the list of questionnaires.
+4. From the "Fetch All Questionnaires" pane, choose the specific questionnaires.
+5. Specify the name. 
+6. Specify the valid for.
+
+- For more details on Survey please refer to the below documentation.
+- https://help.webex.com/en-us/article/nlu4x20/Experience-Management---Interactive-Voice-Response-Surveys-for-Webex-ContactCenter#survey-builder
+- https://help.imiconnect.io/docs/experience-management
+
+### Facebook Close Flow With Survey Node
+
+<br><img width="800" alt="FacebookSurveyCloseFlow" src="../../images/FacebookSurveyCloseFlow.png"><br>
+
+- This flow allows the flow admin to send the survey messages and survey links to the customer over messenger once the agent wraps up the conversation.
+
+-  Pre-requisites :- You must configure the facebook main flow's resolve conversation node with "override default contact close event handling" checkbox being checked.
+
+<br><img width="800" alt="ResolveConversationNode" src="../../images/ResolveConversationNode.png"><br>
+
+-  Set up the survey node. 
+-  In the messenger node,provide survey message and pick the survey link from the previous node as `$(n940.surveyURL)`.
+
+<br><img width="800" alt="MessengerNode" src="../../images/MessengerNode.png"><br>
+
+- In the append conversation node, provide the same survey message and survey link.
+
+<br><img width="800" alt="AppendFacebookConversation" src="../../images/AppendFacebookConversation.png"><br>
+
+- Example of the survey presented to the customer on the messenger.
+
+<br><img width="800" alt="CustomerMessages" src="../../images/CustomerMessages.png"><br>
+
+### LiveChat Close Flow With Survey Node
+
+<br><img width="800" alt="LiveChatSurveyCloseFlow" src="../../images/LiveChatSurveyCloseFlow.png"><br>
+
+- This flow allows the flow admin to send the survey messages and survey links to the customer over chat widget once the agent wraps up the conversation.
+
+-  Pre-requisites :- 
+1. You must configure the livechat main flow's resolve conversation node with "override default contact close event handling" checkbox being checked.
+
+<br><img width="800" alt="ResolveConvNode" src="../../images/ResolveConvNode.png"><br>
+
+2. In the livechat main flow add the set variable node to set threadId and userId
+
+<br><img width="800" alt="SetVariableLiveChat" src="../../images/SetVariableLiveChat.png"><br>
+
+3. Extract the thread and user in the evaluate node of the LiveChatClose flow.
```

Comment:
Can you add the detail why thread and user id are required

Reply:
Sure